krave/ui/constants.py

```diff
@@ -14,13 +14,19 @@ class PATHS:
     TEMP_IMG = '/home/pi/krave/krave/ui/images/graph_analyzed_data.png'
     TEMP_IMG_RESIZED = '/home/pi/krave/krave/ui/images/graph_analyzed_data_resized.png' 
     RUN_TASK = '/home/pi/krave/run_task.sh'
-    INITIAL_COMMUNICATION = '/home/pi/krave/krave/ui/communications/initial_communication.txt'
+    COMMUNICATION = '/home/pi/krave/krave/ui/communications/communication.txt'
+    COMMUNICATION2 = '/home/pi/krave/krave/ui/communications/communication2.txt'
```

Comment:
Consider using `https://docs.python.org/3/library/importlib.resources.html` to find paths to different resource files such as the ones here. This should help if someone were to install the package in a different location than described here. Once again, future fix, no need to change now (maybe add a TODO comment?